client/lib/menu-data/test/test-menu-data.js

```diff
@@ -297,29 +297,6 @@ describe( 'MenuData', function() {
 		} );
 	} );
 
-	describe( 'saveMenu', function () {
```

Comment:
Why did I remove this test instead of fixing it?

I consider it harmful.

- In the whole file wpcom is not properly stubbed
- Test was faililng because `saveMenu` is now async
- Proper async test was impossible because wpcom is not properly stubbed
- Test was testing nothing (a call of some random `interceptSaveForHomepageLink `) at yet it was pretending to test `saveMenu`